client/app/bundles/course/assessment/components/AssessmentForm/translations.ts

```diff
@@ -33,6 +33,12 @@ const translations = defineMessages({
     id: 'course.assessment.AssessmentForm.toggleLiveFeedbackDescription',
     defaultMessage: 'Enable Get Help feature for all programming questions',
   },
+  isCourseCodaveriDisabled: {
+    id: 'course.assessment.AssessmentForm.isCourseCodaveriDisabled',
+    defaultMessage:
+      'Please contact the Course Administrator to enable Codaveri Evaluator\
```

Comment:
There is no terminology "Course Administrator", update and follow [existing example](https://github.com/Coursemology/coursemology2/blob/b9ec3d34bc7cd9f860c8776109c557d477ddf517/client/app/bundles/course/assessment/translations.ts#L1160)

Reply:
done